README.md

```diff
@@ -7,62 +7,53 @@
 
 _Take control of your internal daemons!_
 
-**Pebble** is a lightweight Linux service manager that helps you orchestrate a set of local service processes as an organized set. It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
+**Pebble** is a lightweight Linux service manager that helps you orchestrate a set of local processes as an organised set. It resembles well-known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services. However, it was designed with unique features such as layered configuration and an HTTP API that help with more specific use cases.
 
 Pebble's key features:
 
-- Service management with [layers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)
+- [Layer](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)-based configuration
 - Service [dependencies](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/service-dependencies/)
 - Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/logs/) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
 - [Health checks](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/health-checks/)
-- [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/)
+- [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/) (aggregated events)
 - Identities
-- Can be used in Machines and containers
-- [CLI](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/cli-commands/), [HTTP API and Go client](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/)
+- Can be used in machines and [containers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/pebble-in-containers/)
```

Comment:
Per Tony's earlier suggestion, "VMs" or "virtual machines"

```suggestion
- Can be used in virtual machines and [containers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/pebble-in-containers/)
```

Reply:
Technically it can be used in physical machines as well, that is why I used "machines" in the first place. But physical machines already come with systemd, Pebble probably won't be used there, although it can be. Changing to "virtual machines" then.